powershell/about_XenServer.help.txt

```diff
@@ -45,112 +45,112 @@ The cmdlets fall into the following categories:
 1.  Class Getters
 
     These retrieve a XenAPI object and have names such as Get-XenT, where T is an
-    exposed XenAPI class. The object to get can be specified by ‐Ref or, for those
-    that have a uuid or name, ‐Name or ‐Uuid. If no parameters are specified, all
+    exposed XenAPI class. The object to get can be specified by -Ref or, for those
+    that have a uuid or name, -Name or -Uuid. If no parameters are specified, all
     objects of this type are returned. Example:
 
-        PS> Get‐XenHost ‐Name "Demo Host"
+        PS> Get-XenHost -Name "Demo Host"
 
 2.  Constructors
 
     These create a new XenAPI object and have names such as New-XenT, where T is
     an exposed XenAPI class. Example:
 
-        PS> $vm = Get‐XenVM ‐Name "Demo VM"
-        PS> New‐XenVBD ‐VM $vm ‐VDI $null ‐Userdevice 3 ‐Bootable $false ‐Mode RO
-            ‐Type CD ‐Unpluggable $true ‐Empty $true ‐OtherConfig @{}
-            ‐QosAlgorithmType "" ‐QosAlgorithmParams @{}
+        PS> $vm = Get-XenVM -Name "Demo VM"
+        PS> New-XenVBD -VM $vm -VDI $null -Userdevice 3 -Bootable $false -Mode RO `
+            -Type CD -Unpluggable $true -Empty $true -OtherConfig @{} `
+            -QosAlgorithmType "" -QosAlgorithmParams @{}
 
 3.  Class Removers
 
     These destroy a XenAPI object and have names such as Remove-XenT, where T is
     an exposed XenAPI class.  To specify the object to remove use the parameter
-    ‐T, where T is the exposed XenAPI class, or ‐Ref or, for those objects that
-    have a uuid or name, ‐UUID or ‐Name. Example:
+    -T, where T is the exposed XenAPI class, or -Ref or, for those objects that

```

Comment:
I notice some changes around hyphens. Several symbols exist in UTF8 but not in Ascii, where we just have the minus, which is used for everything. Code that is pasted into the shell use the minus as otherwise it would not be recognised despite looking correct.

Reply:
Indeed, the old "hyphens" were in reality this character in UTF-16: \u00e2\u0080\u0090. I've turned them to minus.